### ISSUE TITLE
[BUG]Atom.jl can not update to the latest version

### ISSUE THREAD

**Issue by shilu1984** [JunoLab/Juno.jl#509]
Please search existing issues to avoid duplicates.

## Details
- Atom version: 1.41.0
- Julia version: 1.3.1
- OS: Windows 10 pro x64
- Package versions:
  - Atom.jl: 0.8.8
  - julia-client: 0.12.2
  - ink:0.12.2

### Steps to reproduce

the version of Atom.jl on my PC is fixed at version 0.8.8, it cannot upgrade to the latest verion with ']up Atom', which makes Juno not work. 
']up Atom' only indicates that there is no change, and there is no error
 I don't know if there is something dependencies wrong with the packages I installed.
![image](https://user-images.githubusercontent.com/37612876/75475676-29f3be00-59d4-11ea-8951-692e36f5ad69.png)

**Comment by pfitzseb** [comment]
Try `pkg> add Atom@0.12.7`. That should tell you which package is stopping the upgrade.

**Comment by shilu1984** [comment]
Many thanks, I have learned a trick.
It is JuliaInterpreter 0.7.8 that block the upgrade of Atom. The problem is solved by upgrade JuliaInterpreter 0.7.8 to the latest with `pkg> add JuliaInterpreter@0.7.12`
It's strange that I used to be able to update all the packages to the latest with `]up`, but this time it failed.

(v1.3) pkg> add Atom@0.12.7
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Atom [c52e3926]:
 Atom [c52e3926] log:
 ├─possible versions are: [0.8.0-0.8.8, 0.9.0-0.9.1, 0.10.0-0.10.2, 0.11.0-0.11.3, 0.12.0-0.12.7] or uninstalled
 ├─restricted to versions 0.12.7 by an explicit requirement, leaving only versions 0.12.7
 └─restricted by compatibility requirements with JuliaInterpreter [aa1ae85d] to versions: [0.8.6-0.8.8, 0.9.0-0.9.1, 0.10.0-0.10.2, 0.11.0-0.11.3] or uninstalled — no versions left
   └─JuliaInterpreter [aa1ae85d] log:
     ├─possible versions are: [0.1.1, 0.2.0-0.2.1, 0.3.0-0.3.2, 0.4.0-0.4.1, 0.5.0-0.5.2, 0.6.0-0.6.1, 0.7.0-0.7.12] or uninstalled
     └─restricted to versions 0.7.8 by an explicit requirement, leaving only versions 0.7.8
